frontend/src/app/profile/profile-editor/profile-editor.component.ts

```diff
@@ -21,6 +21,7 @@ export class ProfileEditorComponent implements OnInit {
   };
 
   public profile: Profile;
+  public token = new String();
```

Comment:
Thinking we can initialize this in the constructor and simplify a few things. Additional push coming for your review.